src/_data/donations.json

```diff
@@ -1,8 +1,17 @@
 [
+    {
+        "id": "SA_kwHOAFvahM4AExPl",
+        "name": "Charlie",
+        "image": "https://avatars.githubusercontent.com/u/38315638",
+        "url": "https://charlie.icu",
+        "amount": 1,
+        "date": "2024-12-30T11:40:47Z",
+        "source": "github"
+    },
     {
         "id": "SA_kwHOAFvahM4AEwvn",
         "name": "Felipe OG",
-        "image": "https://avatars.githubusercontent.com/u/17603069?u=d003c6edefc6fdc4327f7837208de0055e8f949f&v=4",
+        "image": "https://avatars.githubusercontent.com/u/17603069",
```

Comment:
Any idea what `v=4` and `u=d03...` parameters represent and whether it's safe to remove them? I couldn't find any docs about `avatars.githubusercontent.com`.

Reply:
I'm pretty sure that `v=4` is just the GitHub API version used to retrieve the URL. I believe, but I'm not sure, that `u` is a cache key so that if a user changes their avatar, the UI won't be displaying an old one.

Reply:
Ok, then let's try with the simplest format.